src/NosCore.GameObject/Character.cs

```diff
@@ -96,9 +102,13 @@ public bool IsFriendListFull
         public bool Camouflage { get; set; }
 
         public bool Invisible { get; set; }
+
         public IInventoryService Inventory { get; set; }
+
         public bool InExchangeOrTrade { get; set; }
```

Comment:
same here

Reply:
can we also split them into 2 bool and make InExchangeOrTrade => bool1 & bool2
 (use binary &)